src/components/HeroContainer.js

```diff
@@ -91,7 +101,7 @@ const HeroContainerComponent = props => {
         window.removeEventListener('resize', handleWindowSizeChange)
       }
     }
-  }, [sdkRef.current])
+  }, [isSDK, sdkRef])
```

Comment:
I think putting ref here may not be necessary

Reply:
It avoid the es-lint warning